themes/ffrn/layouts/partials/footer.html

```diff
@@ -1,49 +1,51 @@
-                <li><a target="_blank" href="https://wiki.ffrn.de">Wiki</a></li>
-                <li><a target="_blank" href="http://fw.freifunk-rhein-neckar.de">Firmware Download</a></li>
-                <li><a target="_blank" href="https://register.freifunk-rhein-neckar.de/">Knotenverwaltung</a></li>
-            </ul>
-        </div>
-        <div class="col-sm-4 footer-links footer-social">
-            <h4>Social & Kontakt</h4>
-            <hr>
-            <ul>
-                <li><a target="_blank" href="https://forum.ffrn.de/"><i class="fab fa-discourse"></i> Forum</a></li>
-                <li><a target="_blank" href="https://matrix.to/#/#ffrn:ffrn.de"><i class="fa fa-comments"></i> #ffrn:ffrn.de</a> (<a href="http://chat.ffrn.de/">Matrix-Chat</a>)</li>
-                <li><a target="_blank" href="https://twitter.com/FFRheinNeckar"><i class="fab fa-twitter"></i> Twitter</a></li>
-                <li><a target="_blank" href="https://chaos.social/@ffrn" rel="me"><i class="fab fa-mastodon"></i> Mastodon</a></li>
-                <li><a target="_blank" href="https://github.com/Freifunk-Rhein-Neckar"><i class="fab fa-github"></i> GitHub</a></li>
-            </ul>
+        <div class="row">
+            <div class="col-sm-4 footer-links">
+                <h4>Info</h4>
+                <hr>
+                <a href="mailto:info@freifunk-rhein-neckar.de">info@freifunk-rhein-neckar.de</a><br>
+                <a href="/impressum/">Impressum</a>
+                <hr>
+                <address>
+                    <strong>Freifunk Rhein-Neckar e.V.</strong><br>
+                    c/o Jakob Haufe<br>
+                    Hammelbächerstraße 35<br>
+                    D-69469 Weinheim<br>
+                </address>
+                <hr>
+            </div>
+            <div class="col-sm-4 footer-links">
+                <h4>Links</h4>
+                <hr>
+                <ul>
+                    <li><a target="_blank" href="https://freifunk-rhein-neckar.github.io/ffrn/mitgliedsantrag/Mitgliedsantrag.pdf">Mitgliedsantrag</a></li>
+                    <li><a target="_blank" href="https://freifunk-rhein-neckar.github.io/ffrn/satzung/satzung_ffrn.pdf">Vereinssatzung</a></li>
+                    <li><a target="_blank" href="https://freifunk-rhein-neckar.github.io/ffrn/beitragsordnung/beitragsordnung_ffrn.pdf">Beitragsordnung</a></li>
+                    <li><a target="_blank" href="http://map.freifunk-rhein-neckar.de/">Map</a></li>
+                    <li><a target="_blank" href="https://wiki.ffrn.de">Wiki</a></li>
+                    <li><a target="_blank" href="http://fw.freifunk-rhein-neckar.de">Firmware Download</a></li>
+                    <li><a target="_blank" href="https://register.freifunk-rhein-neckar.de/">Knotenverwaltung</a></li>
+                </ul>
+            </div>
+            <div class="col-sm-4 footer-links footer-social">
+                <h4>Social & Kontakt</h4>
```

Comment:
`&amp;`

Reply:
Warum? Ich sehe gerade nicht den Vorteil drin?

Reply:
Grad nachgeschaut, stimmt, solang es nicht zu einem Sonderzeichen erweitert wird sollte es problemfrei sein, sind meist dann nur dir Linter die rumprangern.

Reply:
Linting in der CI wäre ggf. auch noch was. Bin mir nur nicht sicher ob man sich damit nicht mehr Probleme einfängt als das man löst.

Reply:
Das fehlende DIV oben z.B. hab ich schneller über xmllint im HTML-Modus gefunden als im Browser, wo es nur rot markiert wird und mir nicht aufgefallen ist.